Casks/font/font-n/font-nacelle.rb

```diff
@@ -0,0 +1,32 @@
+cask "font-nacelle" do
+  version "1.00"
+  sha256 "4e21fc1e22b15c4e1a7c520cf48ec5d47a5803beee21ac7edc0b205c340a4065"
+
+  url "https://dotcolon.net/files/fonts/nacelle_#{version.to_s.tr(".", "")}.zip"
+  name "Nacelle"
+  homepage "https://dotcolon.net/fonts/nacelle/"
+
+  livecheck do
+    url :homepage
+    regex(/Version[^\d]*(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/Version\s+v?(\d+(?:\.\d+)+)/i)
```

This brings the regex in line with our usual pattern.

Reply:
Makes sense, thank you.
Let's use the corresponding metacharacter for simplicity. I've updated the suggestion accordingly.